src/widgets/Tabs/PatientEdit.js

```diff
@@ -55,17 +64,21 @@ const PatientEditComponent = ({
   canEditPatient,
 }) => {
   const { pageTopViewContainer } = globalStyles;
+  const [isDeceasedModalOpen, toggleIsDeceasedAlert] = useToggle(false);
   const formRef = useRef(null);
-
   const savePatient = useCallback(
     e => {
       updatePatientDetails(completedForm);
-      formRef?.current?.submit(e);
-
-      if (canSaveForm) {
-        onCompleted();
+      if (!completedForm.isDeceased) {
+        formRef?.current?.submit(e);
```

Comment:
Right now if I set a patient to deceased and press next, the isDeseased checks runs before canSaveForm validation, so I think editing user would not be abled to set someone to deceased and save it from mobile.

We have to validate, allow save and check isDeseased afterwards I think.

Reply:
The "Vaccination" process is used specifically for dispensing the vaccines. If user wants to edit any details of the patients they can do that in "Dispensing" -> clicking "Edit" button for the selected patient. I think it is a reasonable limitation not to be bale to set "Deceased" status during the vaccination process.

The behaviour in this PR is consistent with what was before -> if you don't finish the vaccination process and exit it by clicking "Cancel" then all the changes made to the patient details will not be saved.
To reproduce:
- Go to "Vaccination" -> Select an existing patient editable on the current site (let's say "Patient 1")
- Edit some details and click "Next"
- Exit vaccination process by clicking "Exit"
- Go to "Dispensing" -> select "Edit" Patient 1.
- See that details you were editing in vaccination window are not saved because you exited the process with the "Exit" button. 

In my opinion, it is the correct behaviour as clicking "Cancel" always discards any changes made.

Reply:
Also when we try to create new Patient, deceased that is.

Reply:
@chastichno you are right about the form not saving the changes in step 2 if admin presses cancel at point. That is a very strange flow as there is no user feedback so I kinna want an alert here too during cancel to tell user that the data they have entered so far would be discarded, but then I do not want too much alerts either.

Since we cannot edit deceased status in this form anyway, I kinda wanna disabled this option and make it viewable only.

But considering the flow you just described the code works fine as it is. Maybe we can merge it for now and make updates later if there is user feedback and demands.

Reply:
all sounds reasonable to me. The easiest solution would be to make another issue which disables editing of a patient deceased status during the vaccination dispensing process. Like you say Nastia, that's a strange workflow anyway